config/routes.rb

```diff
@@ -87,6 +87,7 @@
 
   resources :user_journeys, path: 'moje-zivotne-situacie' do
     post :restart, on: :member, path: 'zacat-odznova'
+    get :index, on: :collection, path: 'rozpracovane'
```

Comment:
Dajme to ako default index, bez path. Mozno, ze casom tam budu aj tie skoncene ZS. Cize toto tu vlastne netreba, len pouzit user_journeys_path